integrated_channels/utils.py

```diff
@@ -136,3 +136,16 @@ def strfdelta(tdelta, fmt='{D:02}d {H:02}h {M:02}m {S:02}s', input_type='timedel
             values[field], remainder = divmod(remainder, constants[field])
 
     return f.format(fmt, **values)
+
+
+def get_image_url(content_metadata_item):
+    """
+    Return the image URI of the content item.
+    """
+    image_url = ''
+    if content_metadata_item['content_type'] == 'program':
+        image_url = content_metadata_item.get('card_image_url')
```

Comment:
probably not a big deal, but figured i would ask: is it possible (and do we want it to be) that this function can return None and not empty string if the `card_image_url` or `image_url` fields don't exist on the dict for some reason?

Reply:
@christopappas there is some logic around None ... 
https://github.com/edx/edx-enterprise/blob/a1974db6be53da3e4a11265de24d3a6f0b1e6824/integrated_channels/integrated_channel/exporters/content_metadata.py#L138

Reply:
@irfanuddinahmad cornerstone does not accept values with `None` so we are skipping those items where we have a None value.

Reply:
Thanks @ziafazal